### PR TITLE
refactor:tokenprovider and table datetime

### PR DESCRIPTION
**Pull request by MayueCif**

### CODE REVIEW

File: src/Services/Masa.Auth.Service.Admin/Domain/Permissions/Services/PermissionDomainService.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Auth.Service.Admin.Domain.Permissions.Services;
+
+public class PermissionDomainService : DomainService
+{
+
```
Comment (doddgu) [inline src/Services/Masa.Auth.Service.Admin/Domain/Permissions/Services/PermissionDomainService.cs:8]: Remove the blank line.

File: src/Services/Masa.Auth.Service.Admin/Infrastructure/EntityConfigurations/Subjects/UserEntityTypeConfiguration.cs [hunk 2]
```diff
@@ -8,6 +8,8 @@ public class UserEntityTypeConfiguration : IEntityTypeConfiguration<User>
     public void Configure(EntityTypeBuilder<User> builder)
     {
         builder.HasKey(u => u.Id);
+        builder.Property(c => c.Account).UseCollation("SQL_Latin1_General_CP1_CS_AS");
```
Comment (doddgu) [inline src/Services/Masa.Auth.Service.Admin/Infrastructure/EntityConfigurations/Subjects/UserEntityTypeConfiguration.cs:11]: This is for SQL Server, if need to switch to mysql how find it and change it?

File: src/Services/Masa.Auth.Service.Admin/Program.cs [hunk 3]
```diff
@@ -140,7 +141,7 @@
 builder.Services.AddOidcCache(redisConfigOption);
 await builder.Services.AddOidcDbContext<AuthDbContext>(async option =>
 {
-    await option.SeedStandardResourcesAsync();
+    //await option.SeedStandardResourcesAsync();
```
Comment (doddgu) [inline src/Services/Masa.Auth.Service.Admin/Program.cs:144]: ?

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Services/Masa.Auth.Service.Admin/Program.cs (hunk 3 above)
Comment (MayueCif) [reply src/Services/Masa.Auth.Service.Admin/Program.cs:144]: Temporary note,some issues fix for wu

File: src/Services/Masa.Auth.Service.Admin/Infrastructure/EntityConfigurations/Subjects/UserEntityTypeConfiguration.cs (hunk 2 above)
Comment (MayueCif) [reply src/Services/Masa.Auth.Service.Admin/Infrastructure/EntityConfigurations/Subjects/UserEntityTypeConfiguration.cs:11]: mysql should replace 
`e.Property(p => p.CollationColumnFA).ForMySQLHasCollation("utf8_bin"); // defining collation in a property`
whether remove db limit ,change code logic limit?

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**